cirq-rigetti/tests/bell_circuit_test.py

```diff
@@ -0,0 +1,71 @@
+from typing import cast
```

Comment:
Cirq's convention is not to put the tests in a `tests` directory; but along with the source. I prefer this convention because I  read tests as I do documentation and it's too much of a hassle to go into a different directory.